planner/core/planbuilder.go

```diff
@@ -983,6 +983,17 @@ func buildShowDDLJobsFields() *expression.Schema {
 	return schema
 }
 
+func buildTableRegionsSchema() *expression.Schema {
+	schema := expression.NewSchema(make([]*expression.Column, 0, 10)...)
+	schema.Append(buildColumn("", "REGION_ID", mysql.TypeLonglong, 4))
+	schema.Append(buildColumn("", "START_KEY", mysql.TypeVarchar, 64))
```

Comment:
Will the string length large than 64?

Reply:
Currently, it is ok. `string length` doesn't matter here.